### ISSUE TITLE
API Access to Search Results

### ISSUE THREAD

**Issue by timgdavies** [OpenDataServices/org-ids#46]
Users should be able to send queries to an API, and get back a JSON representation of a list of lists, with ranking already applied.

**Comment by andylolz** [comment]
Not sure how the Aidstream work is possible without this? Unless they’re recreating the ranking algorithm?

**Comment by timgdavies** [comment]
Good point - I think they may still be working on mock-up stage, so are going to hit a requirement for this pretty soon. I don't think it would be too tricky to implement.
